### DISCUSSION TITLE
Start with first tab in layout

### DISCUSSION THREAD

**Discussion by ciaransweet** [zellij-org/zellij#2103] [Q&A] [answered]
Hey folks, in the following example:

```kdl
// ~/.config/zellij/layouts/example.kdl
layout {
    tab name="Tab 1" {
        pane size=2 borderless=true {
            plugin location="zellij:tab-bar"
        }
        pane name="Pane 1" {}
    }
    tab name="Tab 2" {
        pane size=2 borderless=true {
            plugin location="zellij:tab-bar"
        }
        pane name="Pane 2" {}
    }
}
```

I get `Tab 1` and `Tab 2` in that order, however when I open `zellij --layout example`, `Tab 2` is selected as the active tab - Is there a way to have the first tab in the list the active tab on startup?

https://user-images.githubusercontent.com/9111975/213441085-48df194c-9355-44fa-a976-d1c093479a07.mov

**Comment by ciaransweet** [comment]
Doh. Just seen https://zellij.dev/documentation/creating-a-layout.html?highlight=tab#focus-1

**Reply by ciaransweet** [reply]
It seems that `focus` doesn't affect this.

**Reply by ciaransweet** [reply]
If I use:

```
layout {
    tab name="Tab 1" focus=true {
        pane size=2 borderless=true focus=false {
            plugin location="zellij:tab-bar"
        }
        pane name="Pane 1" focus=false {}
    }
    tab name="Tab 2" focus=false {
        pane size=2 borderless=true focus=false {
            plugin location="zellij:tab-bar"
        }
        pane name="Pane 2" focus=false {}
    }
}
```

I still start with `Tab 2` in focus..

**Comment by imsnif** [comment, marked as answer]
Hey, this is a bug: https://github.com/zellij-org/zellij/issues/2021

Hoping to get this fixed by next version

**Reply by ciaransweet** [reply]
Ah nice one @imsnif !
